include/irrString.h

```diff
@@ -913,46 +923,26 @@ class string
 };
 
 
-//! Typedef for character strings
-typedef string<c8> stringc;
-
-//! Typedef for wide character strings
-typedef string<wchar_t> stringw;
-
 //! Convert multibyte string to wide-character string
 /** Wrapper around mbstowcs from standard library, but directly using Irrlicht string class.
 What the function does exactly depends on the LC_CTYPE of the current c locale.
 \param destination Wide-character string receiving the converted source
 \param source multibyte string
-\return The number of wide characters written to destination, not including the eventual terminating null character or -1 when conversion failed */
-static inline size_t multibyteToWString(string<wchar_t>& destination, const core::string<c8>& source)
-{
-	return multibyteToWString(destination, source.c_str(), (u32)source.size());
-}
+\return The number of wide characters written to destination, not including the eventual terminating null character or -1 when conversion failed
 
-//! Convert multibyte string to wide-character string
-/** Wrapper around mbstowcs from standard library, but directly writing to Irrlicht string class.
-What the function does exactly depends on the LC_CTYPE of the current c locale.
-\param destination Wide-character string receiving the converted source
-\param source multibyte string
-\return The number of wide characters written to destination, not including the eventual terminating null character  or -1 when conversion failed. */
-static inline size_t multibyteToWString(string<wchar_t>& destination, const char* source)
+This function should not be used and is only kept for "CGUIFileOpenDialog::pathToStringW". */
```

Comment:
can we maybe just drop this function?
or wait for https://github.com/minetest/minetest/pull/13512 so we can drop the class entirely...

Reply:
> can we maybe just drop this function?

How? If it is used?

> or wait for https://github.com/minetest/minetest/pull/13512 so we can drop the class entirely...

I'd like to avoid that, it could take a while until minetest/minetest#13512 is merged.

Reply:
You're right, we can always get rid of it later.

I did actually want to pick up https://github.com/minetest/minetest/pull/13512 and fix it because it's a good an necessary change, but I'm totally short on time right now.